requirements-dev.txt

```diff
@@ -59,3 +59,4 @@ nvidia-ml-py
 huggingface_hub
 pyamg>=5.0.0
 git+https://github.com/Project-MONAI/GenerativeModels.git@7428fce193771e9564f29b91d29e523dd1b6b4cd
+git+https://github.com/facebookresearch/segment-anything.git@6fdee8f2727f4506cfbbe553e23b895e27956588
```

Comment:
Hi @KumoLiu ,

Should we add all the required places for this new optional dependency?
Referring to: https://github.com/Project-MONAI/MONAI/blob/dev/CONTRIBUTING.md#adding-new-optional-dependencies.

Thanks.

Reply:
I try to add it in the setup.cfg.  Other place should be ok.